python/ray/data/context.py

```diff
@@ -114,6 +116,7 @@
     "AWS Error NETWORK_CONNECTION",
     "AWS Error SLOW_DOWN",
     "AWS Error UNKNOWN (HTTP status 503)",
+    "AWS Error SERVICE_UNAVAILABLE",
```

Comment:
should we also add this to `DEFAULT_RETRIED_IO_ERRORS`?

Reply:
It is in it. You might need to expand the diff
<img width="362" alt="image" src="https://github.com/user-attachments/assets/c7f90638-bf5f-4ede-8d14-8361dfd64f08">

Reply:
oh sorry i got confused by the top comment above `DEFAULT_WRITE_FILE_RETRY_ON_ERRORS`. for backwards compatibility, should we also add it in `DEFAULT_WRITE_FILE_RETRY_ON_ERRORS`?

Reply:
`DEFAULT_WRITE_FILE_RETRY_ON_ERRORS` isn't used anywhere right now. If you try to configure `write_file_retry_on_errors`, Ray Data emits a warning instructed you to configure `retried_io_errors` instead.